### DISCUSSION TITLE
Section `macOS > Browser behavior` does not list information about Firefox

### DISCUSSION THREAD

**Discussion by foss-** [passkeydeveloper/community#4] [Passkeys on the Web Q&A] [answered]
https://passkeys.dev/docs/reference/macos/#browser-behavior does not list information about Firefox. Firefox does support and work with Passkeys on macOS. Are there plans to add information?

**Comment by timcappalli** [comment, marked as answer]
@foss- the reference page sections about browsers were designed to highlight known limitations. As current shipping versions of Firefox do not have any passkey-specific limitations, it is not listed.

**Comment by foss-** [comment]
Interesting, thanks for elaborating.
So should https://bugzilla.mozilla.org/show_bug.cgi?id=1862132 be listed as limitation on https://passkeys.dev/docs/reference/android/?

**Reply by timcappalli** [reply]
It doesn't really affect the way a developer adds support for passkeys to their site so we don't typically include stuff like this. But if you think it's important for a developer to know, then we can add it. Let me know or feel free to create a PR.

**Reply by foss-** [reply]
It is fine, still trying to get an understanding of what the websites purpose is, so your answers are helpful and appreciated.

**Reply by timcappalli** [reply]
The site is focused on helping developers add support for passkeys to their service.
